client/blocks/eligibility-warnings/hold-list.tsx

```diff
@@ -85,10 +86,8 @@ function getHoldMessages(
 			supportUrl: null,
 		},
 		EXCESSIVE_DISK_SPACE: {
-			title: translate( 'Upload not available' ),
-			description: translate(
-				'This site is not currently eligible to install themes and plugins. Please contact our support team for help.'
-			),
+			title: translate( 'Meet disk quota' ),
```

Comment:
This might benefit from a translator comment with some more context.

Reply:
[Like this?](https://github.com/Automattic/wp-calypso/blob/54425b90730d37be961c94af3d6f799ebbda4189/client/blocks/eligibility-warnings/hold-list.tsx#L89)

Reply:
The `i18n-calypso` package provides an optional `comment` argument for doing this. See https://github.com/Automattic/wp-calypso/tree/trunk/packages/i18n-calypso#options.

I think we can also note something related to the Atomic eligibility warnings.

```suggestion
			title: translate( 'Meet disk quota', { comment: 'Message displayed when a Simple site cannot be transferred to Atomic because there is not enough disk space. It appears inside a list with actions to perform in order to proceed with the transfer.'} ),
```